addons/portal/views/portal_templates.xml

```diff
@@ -7,6 +7,7 @@
         </xpath>
         <xpath expr="//div[@id='wrapwrap']/header/img" position="replace">
             <t t-cache="res_company">
+                <a class="o_skip_to_content btn btn-primary rounded-0 visually-hidden-focusable position-absolute" href="#wrap" tabindex="1" groups="!base.group_user">Skip to Content</a>
```

Comment:
#### 1) Tabindex
The tabindex should not be set on 1. I understand the idea and that seems ok at first, but there might be some cases that don't fit that mold: in particular, the cookies bar should be the actual 1st focusable element on the page. (NB: that's not the case for now. But setting `tabindex="1"` here would prevent the cookies bar from any possibility to capture the focus, as modals are added at the end of the DOM.)
`tabindex="2"` would probably be good enough, while leaving room for other things to bypass the priority if absolutely needed.

#### 2) Skip link in header
I don't think the skip link should be set inside the header, for several reasons:
- from a DOM structure point of view, it's not really part of the header.
- with sticky headers, there is no real reason to keep the skip link sticky too. That should just be a help for those who want/need it. But if you tab, then scroll, you probably don't want to keep the button over the header.
- (+ if we finally want to set a z-index > modals, that should be outside the header to escape its stacking context.)

So it should at least be placed before the `<header>`, and possibly even before the `#wrapwrap`.